libs/libc/string/lib_bzero.c

```diff
@@ -0,0 +1,40 @@
+/****************************************************************************
+ * libs/libc/string/lib_bzero.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+#include <sys/types.h>
+#include <string.h>
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: bzero
+ ****************************************************************************/
+
+void bzero(FAR void *s, size_t n)
```

Comment:
For `memset` we have
```
#undef memset /* See mm/README.txt */
no_builtin("memset")
```
do we need anything similar here to solve issue discussed in https://github.com/apache/nuttx/pull/11207 ?

Reply:
i try
```
#undef bzero /* See mm/README.txt */
no_builtin("bzero")
```
but not useful